src/app/header/menu/menu.component.scss

```diff
@@ -112,25 +116,27 @@ ul {
       }
 
       &.sectioned {
-        width: auto;
-        min-width: 500px;
         display: flex;
-        margin-left: -$menu-section-width;
+        flex-direction: column;
+        margin-left: calc(#{$menu-dropdown-sectioned-width} / -2);
```

Comment:
Same here too, size can be achieved without calc: `$menu-dropdown-sectioned-width / -2`